Pseudo_experiments/nbconverted/create_heatmap.py

```diff
@@ -88,15 +96,17 @@
     "output_control",
     "DE_stats_control_data_"+experiment_id+"_example.txt")
 
+# Gene number to gene name file
+gene_name_file = os.path.join(
+    base_dir,
+    "Pseudo_experiments",
+    "Pseudomonas_aeruginosa_PAO1_107.csv")
```

Comment:
Is it okay that this is hardcoded? Seems like something that might be different per each file run, but maybe not.

Reply:
This is something that can be adjusted by the user depending on the annotations they want to use. Since this isn't part of the main set of scripts, I think its ok to have in the top set of cells so that a user can manually change if needed.